application/account-management/WebApp/.prettierrc

```
@@ -0,0 +1,8 @@
+{
+    "printWidth": 80,
```

This conflicts with the setting in .editorconfig, which says that the break is 120 characters. I know that 80 characters is the "default" from back in the days, when we had small screens. In C# the default is 120 characters, which IMO is much better, as it will give more vertical screen real estate.
```suggestion
    "printWidth": 120,
```
This conflicts with the setting in .editorconfig, which says that the break is 120 characters. I know that 80 characters is the "default" from back in the days, when we had small screens. In C# the default is 120 characters, which IMO is much better, as it will give more vertical screen real estate.